pick.c

```diff
@@ -649,12 +608,72 @@ tty_init(void)
 
 	setupterm((char *)0, fileno(tty_out), (int *)0);
 
-	choices_lines = lines - 1;	/* available lines, minus query line */
+	tty_size();
 
 	if (use_alternate_screen)
 		tty_putp(enter_ca_mode, 0);
 
 	signal(SIGINT, handle_sigint);
+#if HAVE_DECL_SIGWINCH
+	signal(SIGWINCH, handle_sigwinch);
+#endif
+}
+
+void tty_draw(void)
+{
+	size_t	i, j;
+	size_t	xscroll = 0;
+
+	tty_putp(cursor_invisible, 0);
+	tty_putp(cursor_home, 0);	/* move cursor home to avoid screen clutter */
```

Comment:
What kind of clutter appears if we don't output this? I tried removing this line but could reproduce any issues.